instagram/config/routes.rb

```diff
@@ -0,0 +1,60 @@
+Rails.application.routes.draw do
+
+  resources :pictures do
+    
+  end
+  # The priority is based upon order of creation: first created -> highest priority.
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)